lib/restforce/collection.rb

```diff
@@ -31,6 +31,11 @@ def size
     end
     alias_method :length, :size
 
+    # Return whether the collection contains any objects.
```

Comment:
I'd be quite keen to stick with what we have now rather than adding an alias - potentially something you could add as a monkey patch if you like? Or would `#none?` serve your needs better? (At least avoids a `!`...)